dockstore-webservice/src/main/java/io/dockstore/webservice/resources/proposedGA4GH/ToolsApiExtendedServiceImpl.java

```diff
@@ -657,7 +658,13 @@ public Response updateAITopic(String id, UpdateAITopicRequest updateAITopicReque
         }
         checkEntryNotNull(entry);
 
+        Optional<? extends Version<?>> versionOptional = getVersion(entry, version);
+        if (versionOptional.isEmpty()) {
+            throw new CustomWebApplicationException(VERSION_NOT_FOUND_ERROR, HttpStatus.SC_NOT_FOUND);
+        }
```

Comment:
There is an `Optional.orElseThrow` method that will return the wrapped value if it exists, and throw otherwise:
https://docs.oracle.com/javase%2F8%2Fdocs%2Fapi%2F%2F/java/util/Optional.html#orElseThrow-java.util.function.Supplier-
Using it, we can reduce this block to something like:
```
Version<?> version = getVersion(entry, versionName).orElseThrow(
    () -> new CustomWebApplicationException(VERSION_NOT_FOUND_ERROR, HttpStatus.SC_NOT_FOUND));
```